src/components/MissingDocAlert.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import {
+    Alert,
+    Box,
+    Button,
+    CloseButton,
+    Flex,
+    Heading,
+    Link,
+    Text,
+} from "@chakra-ui/react";
+import colourTheme from "@styles/colours";
+import InfoIcon from "@components/icons/InfoIcon";
+import { roles } from "@prisma/client";
+import { UseMeResponse } from "@utils/useMe";
+
```

Comment:
let's use `next/link` for redirects instead
```suggestion
import Link from "next/link";
```